corehq/apps/ota/views.py

```diff
@@ -328,3 +328,16 @@ def get_next_id(request, domain):
     if bucket_id is None:
         return HttpResponseBadRequest("You must provide a pool_id parameter")
     return HttpResponse(SerialIdBucket.get_next(domain, bucket_id, session_id))
+
+
+# Note: this endpoint does not require authentication
+@location_safe
+@require_GET
+def recovery_measures(request, domain, build_id):
+    app_id = get_app_cached(domain, build_id).master_id
```

Comment:
is it worth checking that this `app_id` matches the one in the request?

Reply:
I left it out because the spec explicitly mentions to just pass through without validation:
> app_id → The unique ID of the CC app sending the request (no validation needs to be performed using this. Its sole purpose is to be included in the response; see below)

It does seem like a check would be prudent, though. @amstone326 what do you think?